cliStocksTracker.py

```diff
@@ -652,12 +589,19 @@ def gen_graph(self, auto_colors):
                 color = webcolors.hex_to_rgb(auto_colors[i % 67])
             elif self.colors[i].startswith("#"):
                 color = webcolors.hex_to_rgb(self.colors[i])
-
             else:
                 color = webcolors.hex_to_rgb(
                     webcolors.CSS3_NAMES_TO_HEX[self.colors[i]]
                 )
 
+            # some ticker data returns a NaN value for certain points
+            # instead of removing them, steal your neighbors value so we can graph
+            # iterating all stock data isn't ideal but we can refactor this later
+            for q in range(len(stock.data)):
+                if math.isnan(stock.data[q]):
+                    index = q - 1 if q - 1 >=0 else q + 1
+                    stock.data[q] = stock.data[index]
```

Comment:
see the comment above but this accounts for potential NaN values in ticker data. There should be a more efficient way to do this but I was struggling with syntax. In either regard, this is still faster than the previous implementation of making a request per ticker